core/src/ast_builder/table_name.rs

```diff
@@ -54,11 +54,19 @@ impl<'a> TableNameNode {
     }
 
     pub fn drop_table(self) -> DropTableNode {
-        DropTableNode::new(self.table_name, false)
+        DropTableNode::new(self.table_name, false, false)
     }
 
     pub fn drop_table_if_exists(self) -> DropTableNode {
-        DropTableNode::new(self.table_name, true)
+        DropTableNode::new(self.table_name, true, false)
+    }
+
+    pub fn drop_table_cascade(self) -> DropTableNode {
+        DropTableNode::new(self.table_name, false, true)
+    }
+
+    pub fn drop_table_if_exists_cascade(self) -> DropTableNode {
+        DropTableNode::new(self.table_name, true, true)
```

Comment:
should it be implemented by builder pattern?

Reply:
that sounds nice.
something like these below...?

```rust
drop_table("Foo");
drop_table("Foo").if_exists();
drop_table("Foo").cascade();
drop_table("Foo").if_exists().cascade();

```